cli/.gitignore

```diff
@@ -0,0 +1,47 @@
+# Cargo build directory
+target/
+debug/
+Cargo.lock
```

Comment:
I can put only this line, the rest should be caught by the root .gitignore (can you confirm?)

Reply:
Done and yes I just did `git rm -rf --chached .` and then `git add --all` and `git commit` and this popped up:
```
On branch update-gitignore
Your branch is up to date with 'origin/update-gitignore'.

nothing to commit, working tree clean
```
so looks good to me :+1: